WcaOnRails/app/views/competitions/mycomps.html.erb

```diff
@@ -0,0 +1,62 @@
+<% provide(:title, 'Upcoming competitions') %>
+
+<div class="container">
+  <h2><%= yield(:title) %></h2>
+
+  <table class="table">
```

Comment:
This is what it looks like :P
![2016-01-08-092111_1193x304_scrot](https://cloud.githubusercontent.com/assets/1881933/12196866/4c3c3462-b5e9-11e5-9d94-d20d093216b4.png)